README.md

```diff
@@ -0,0 +1,23 @@
+<h1>Task</h1>
+The task of the project is to create internet shop site for buying products 
+online. Site administrators should have an ability to edit product and user
+lists. Other users should have an ability to add (remove) products to
+shopping cart, complete orders, view and edit their list of orders.
+
+<h1>Technology stack</h1>
+<h3>Server-side:</h3>
+<li>Tomcat</li>
+<li>Java Servlets</li>
+<li>Java 11</li>
+<li>JDBC</li>
+<li>MySQL</li>
+<li>Maven</li>
+<li>log4j</li>
+
```

Comment:
let's add set up guide too.

Reply:
and be sure that you(or somebody else) can see your README.md properly on github.com